connectbd.rb

```diff
@@ -0,0 +1,12 @@
+require 'pg'
+
+def connect_bd_and_exec(sql_command)
+  conn = PGconn.connect('localhost', 5432, nil, nil, 'posts', 'postgres', 'mude6363')
+  res = conn.exec("#{sql_command}")
+  res.each do |r|
```

Comment:
why `.each` is called here? are you sure you need `.each` for a insert/update/delete ?